querybean-generator/src/main/java/io/ebean/querybean/generator/Processor.java

```diff
@@ -1,123 +0,0 @@
-package io.ebean.querybean.generator;
-
-import javax.annotation.processing.AbstractProcessor;
-import javax.annotation.processing.ProcessingEnvironment;
-import javax.annotation.processing.RoundEnvironment;
-import javax.lang.model.SourceVersion;
-import javax.lang.model.element.Element;
-import javax.lang.model.element.TypeElement;
-import java.util.Arrays;
-import java.util.LinkedHashSet;
-import java.util.Set;
-
-/**
- * Process compiled entity beans and generates 'query beans' for them.
- */
-public class Processor extends AbstractProcessor implements Constants {
```

Comment:
Doing the rename here means the DIFF isn't showing now ...

Reply:
should be good now